demo/api.ts

```diff
@@ -28,9 +28,13 @@ export type Pet = {
   name: string;
   photoUrls: string[];
   tags?: Tag[];
+  /*pet status in the store*/
```

Comment:
Can we make those JSDoc Comments like `/** pet status in the store */`?
That way the description will actually be picked up by editors (VSCode at least does not show normal comments, only JSDoc).

<img width="251" alt="image" src="https://github.com/oazapfts/oazapfts/assets/911218/d4cde6ec-0ef8-4876-b782-27f3f14a5e5a">

<img width="251" alt="image" src="https://github.com/oazapfts/oazapfts/assets/911218/f8a41ee7-27b1-456f-b519-2da8b147ead6">

Reply:
Done